simplex.py

```diff
@@ -195,25 +197,20 @@ def run(self, r_data=None):
             ref_ff = self.ff.__class__()
             ref_ff.method = 'REFLECTION'
             ref_ff.params = copy.deepcopy(best_ff.params)
+            # Need score difference sum for weighted inversion. If zero, should break
+            score_diff_sum=sum([x.score - self.new_ffs[-1].score
```

Comment:
I think I will try to use the GitHub editor here, and see what that does. I've never tried it before, so bear with me!